docs/sources/write/reuse-content/reuse-directories/index.md

```diff
@@ -9,32 +9,31 @@ keywords:
   - shared content
   - Hugo
 menuTitle: Reuse directories of content
-review_date: "2024-03-14"
+review_date: "2024-06-26"
 title: Reuse directories of content with Hugo mounts
 ---
 
 # Reuse directories of content with Hugo mounts
 
 {{< admonition type="note" >}}
-The following task can only be performed by a Grafana Labs employee.
+Only a Grafana Labs employee can perform the following task.
 {{< /admonition >}}
 
-Before Hugo performs a build, _Hugo mounts_ mount a source directory at a destination directory.
-Use Hugo mounts to reuse a whole directory of content in another part of the published technical documentation.
+Before Hugo performs a build, [_Hugo module mounts_](https://gohugo.io/hugo-modules/configuration/#module-configuration-mounts) mount a source directory at a destination directory.
```

Comment:
This sentence still reads oddly... should it be `mounts a source directory`?  (even that way it's still sounding odd to me)

Reply:
Yeah I actually spent a bit of time trying to reword it and then left it.

What about:

```suggestion
Before Hugo performs a build, [_Hugo module mounts_](https://gohugo.io/hugo-modules/configuration/#module-configuration-mounts) configure a virtual filesystem.
Each mount mounts a source directory at a destination directory within that filesystem.
```

Reply:
Ah, that makes a LOT more sense.